### PR TITLE
HPCC-14674 Read WU CreateTime/ModifyTime from Statistics

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@richardkchapman please review.

**Comment by richardkchapman** [comment]
@ghalliday please review

File: plugins/workunitservices/workunitservices.ipp [hunk 1]
```diff
@@ -78,13 +114,27 @@ inline bool serializeWUSrow(IPropertyTree &pt,MemoryBuffer &mb, bool isonline)
...
 
-    const char *mod = "TimeStamps/TimeStamp[@application=\"workunit\"]/Modified";
-    const char *crt = "TimeStamps/TimeStamp[@application=\"workunit\"]/Created";
-    fixedAppend(mb,20,pt,crt);
-    if (pt.hasProp(mod))
-        fixedAppend(mb,20,pt,mod);
+    StringBuffer s1, s2;
+    const char* modTime = readModifyTime(pt, s1, true);
+    const char* crtTime = readCreateTime(pt, s2, true);
+    if (crtTime && *crtTime)
```
Comment (ghalliday) [inline plugins/workunitservices/workunitservices.ipp]: Minor:  Possibly simpler to not return a char *, and use the buffer passed in instead.  E.g., if (crtTime.length())

**Comment by ghalliday** [comment]
@richardkchapman Should this be directly accessing the property trees or going via the workunit interface instead?  What affect does moving workunits to dali have on this?

**Comment by richardkchapman** [comment]
Should be ok reading the ptree.

**Pull request merged**